synfig-core/src/synfig/valuenodes/valuenode_atan2.h

```diff
@@ -42,34 +42,27 @@ class ValueNode_Atan2 : public LinkableValueNode
 	ValueNode::RHandle x_;
 	ValueNode::RHandle y_;
 
-	ValueNode_Atan2(const ValueBase &value);
-
 public:
-
 	typedef etl::handle<ValueNode_Atan2> Handle;
 	typedef etl::handle<const ValueNode_Atan2> ConstHandle;
 
-
-	virtual ValueBase operator()(Time t)const;
-
+	static ValueNode_Atan2* create(const ValueBase &value=ValueBase());
+	ValueNode_Atan2(const ValueBase &value);
```

Comment:
I don't know the reason why this constructor was private, maybe it is better to leave it as it was?

Reply:
I don't know either.
Maybe I should label them as `protected` instead of `private`? In case of anyone deriving one of those valuenodes classes?

Reply:
@ice0 ping

Reply:
> Maybe I should label them as protected instead of private? In case of anyone deriving one of those valuenodes classes?

I prefer to keep them private until someone really wants to use them, but ok.

Reply:
Done!